src/javascript/DesignSystem/DatePicker/DatePicker.spec.jsx

```diff
@@ -80,6 +80,19 @@ describe('DatePicker', () => {
         expect(cmp.find('DayPicker').props().disabledDays).toEqual(disabledDays);
     });
 
+    it('should still support selected month', () => {
+        const month = new Date(1972, 11);
+        const cmp = shallowWithTheme(
+            <DatePicker month={month} {...defaultProps}/>,
+            {},
+            dsGenericTheme
+        ).dive();
+
+        const cmpDate = cmp.find('DayPicker').dive().find('Month').dive().find('Day').first().props().day;
```

Comment:
```suggestion
        const cmpMonth = cmp.find('DayPicker').props().month;
        
        expect(cmpMonth).toEqual(month);
```

Simpler test and it take less time to run as you only test your component and not the daypicker external component